rubocop-infinum.gemspec

```
@@ -30,10 +31,11 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency('pry-byebug', '< 4')
   spec.add_development_dependency('rspec', '~> 3.9')
 
-  spec.add_runtime_dependency('rubocop', '>= 1.28.0')
-  spec.add_runtime_dependency('rubocop-factory_bot')
-  spec.add_runtime_dependency('rubocop-performance')
-  spec.add_runtime_dependency('rubocop-rails')
-  spec.add_runtime_dependency('rubocop-rspec')
-  spec.add_runtime_dependency('rubocop-rspec_rails')
+  spec.add_dependency('lint_roller', '~> 1.1')
+  spec.add_dependency('rubocop', '>= 1.72.1', '< 2.0')
+  spec.add_runtime_dependency('rubocop-factory_bot', '>=2.27.1', '< 3.0')
+  spec.add_runtime_dependency('rubocop-performance', '>= 1.24.0', '< 2.0')
+  spec.add_runtime_dependency('rubocop-rails', '>= 2.30.0', '< 3.0')
+  spec.add_runtime_dependency('rubocop-rspec', '>= 3.5.0', '< 4.0')
+  spec.add_runtime_dependency('rubocop-rspec_rails', '>= 2.31.0', '< 3.0')
```

I would remove upper bounds for dependency versions and also the specifier for `lint_roller` (it's not [here](https://docs.rubocop.org/rubocop/plugin_migration_guide.html#2-update-gemspec-file) either):
```suggestion
  spec.add_dependency('lint_roller')
  spec.add_dependency('rubocop', '>= 1.72.1')
  spec.add_runtime_dependency('rubocop-factory_bot', '>=2.27.1')
  spec.add_runtime_dependency('rubocop-performance', '>= 1.24.0')
  spec.add_runtime_dependency('rubocop-rails', '>= 2.30.0')
  spec.add_runtime_dependency('rubocop-rspec', '>= 3.5.0')
  spec.add_runtime_dependency('rubocop-rspec_rails', '>= 2.31.0')
```

I don't think we need upper bounds for dependencies; if a major update of any dependency doesn't break this gem then great because everyone can continue using it, and if it breaks something devs can just not update.

---

Good point! I had some doubts when adding them.  
If we don't have upper bounds, we will only need to update if something breaks.
Updated the code